core/primitives/jwt/jwt.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <jwt-cpp/jwt.h>
```

Comment:
forward declare `jwt::algorithm::hmacsha` (which is used only for shared_ptr) in this header (which is included by "common_api.hpp") and remove this include.

Reply:
move to common/api_secret